### ISSUE TITLE
Mistake in your formula

### ISSUE THREAD

**Issue by vithmel** [Treora/scroll-into-view#2]
I think there is a mistake in your formula.
 
```
var targetY = window.scrollY + rect.top - (window.innerHeight - this.offsetHeight) * options.vertical;
var targetX = window.scrollX + rect.left - (window.innerWidth - this.offsetWidth) * options.horizontal;
```

Because you take window.innerWidth/Height which includes the size of the scrollbar. Maybe better to take:
```
document.documentElement.clientWidth
document.documentElement.clientHeight

var targetY = window.scrollY + rect.top - (document.documentElement.clientHeight - this.offsetHeight) * options.vertical;
var targetX = window.scrollX + rect.left - (document.documentElement.clientWidth - this.offsetWidth) * options.horizontal;
```
I tried to place a div with a height of 10 pixels at the bottom of the page:
`div.scrollIntoView({vertical:1, horizontal: 0})`
, but it was always hidden behind a scrollbar. 
When I changed window.innerWidth / Height to document.documentElement.clientWidth / Height everything started working.

**Comment by Treora** [comment]
Thanks for reporting! You are completely correct. [Apparently](https://drafts.csswg.org/cssom-view/#dom-element-clientheight) there is a small catch with documents rendered in quirks mode, in which case one may need to read `clientWidth`/`clientHeight` from  `document.body` instead of `document.documentElement` (at least in Firefox, in Chromium both give the desired result, and I have not tested other browsers).

**Comment by Treora** [comment]
Fixed in #3, published as v0.1.1.
